packages/client/src/containers/Dashboard/Dashboard.js

```diff
@@ -32,6 +33,9 @@ const Dashboard = () => {
             </div>
           </div>
         </div>
+        <div>
```

Comment:
There is no need to wrap this in an extra `div`, you can just use it as is.
```suggestion
        <TeamMembers />
```